sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/utils/CxxReportSensor.java

```diff
@@ -239,6 +240,21 @@ private boolean saveViolation(Project project, SensorContext context, String rul
           = org.sonar.api.resources.File.fromIOFile(new File(normalPath), project);
```

Comment:
performance needs to be measured...

On Thu, 3 Sep 2015 at 14:52 Günter Wirth <notifications@github.com> wrote:

> In
> sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/utils/CxxReportSensor.java
> <https://github.com/wenns/sonar-cxx/pull/611#discussion_r38637121>:
>
> > @@ -239,6 +240,21 @@ private boolean saveViolation(Project project, SensorContext context, String rul
> >            = org.sonar.api.resources.File.fromIOFile(new File(normalPath), project);
>
> fromIOFile is deprecated, see
> http://javadocs.sonarsource.org/4.5.2/apidocs/deprecated-list.html. So we
> should use something like:
>
> InputFile file= fs.inputFile(fs.predicates().is(new File(filename)));int maxLine = file.lines();
>
> InputFile has method lines.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wenns/sonar-cxx/pull/611/files#r38637121>.
>